### DISCUSSION TITLE
Add suport for client mode in config scanner

### DISCUSSION THREAD

**Discussion by marticardus** [aquasecurity/trivy#6278] [Ideas]
### Description

Currently fs or repo targets, with misconfig scanner in remote server doesn't report anything.

In client mode
```
trivy fs --scanners misconfig --server http://trivy.trivy.svc:8080 . --format json
2024-03-06T09:38:04.516Z	INFO	Misconfiguration scanning is enabled
{
  "SchemaVersion": 2,
  "CreatedAt": "2024-03-06T09:38:18.435793918Z",
  "ArtifactName": ".",
  "ArtifactType": "filesystem",
  "Metadata": {
    "ImageConfig": {
      "architecture": "",
      "created": "0001-01-01T00:00:00Z",
      "os": "",
      "rootfs": {
        "type": "",
        "diff_ids": null
      },
      "config": {}
    }
  }
}
```

In local mode

```
trivy fs --scanners misconfig --server http://trivy.trivy.svc:8080 . --format json
input2024-03-06T09:39:52.714Z	INFO	Misconfiguration scanning is enabled
{
  "SchemaVersion": 2,
  "CreatedAt": "2024-03-06T09:40:06.692953524Z",
  "ArtifactName": ".",
  "ArtifactType": "filesystem",
  "Metadata": {
    "ImageConfig": {
      "architecture": "",
      "created": "0001-01-01T00:00:00Z",
      "os": "",
      "rootfs": {
        "type": "",
        "diff_ids": null
      },
      "config": {}
    }
  },
  "Results": [
(alll results from my scan)
  ]
}
```

### Target

Filesystem

### Scanner

Misconfiguration

**Comment by nikpivkin** [comment]
Hi @marticardus !

Your Trivy run commands are the same in the two examples. Could you provide an example of the config you are scanning? I checked locally and Trivy in client mode scanned the config successfully.

**Reply by marticardus** [reply]
Hi @nikpivkin,

Sorry, the first command is without "--server http://trivy.trivy.svc:8080" (copy&paste error), i was tested with a Dockerfile in directory scanned with "trivy fs"


For example a Dockerfile with "FROM php:5.6-fpm-stretch" (really too old and vulnerable today)

**Comment by marticardus** [comment]
It seems i testing with diferent versions of trivy in local and remote.
